plugins/queue/scheduler.js

```diff
@@ -267,19 +267,34 @@ async function start(executor, config) {
     timeOutOfWindows(freezeWindows, currentTime);
 
     let enq;
+    const buildStats = build && build.stats;
+
+    // for backward compatibility
+    if (buildStats) {
+        // need to reassign so the field can be dirty
+        build.stats = merge(build.stats, {
+            queueEnterTime: new Date().toISOString()
+        });
+    }
 
     // Check freeze window
     if (currentTime.getTime() > origTime.getTime() && !forceStart) {
+        const payload = {
```

Comment:
let

Reply:
reverting to const, as linter complains.